Resources/js-website/src/services/requester.js

```diff
@@ -1,66 +1,103 @@
 import request from '../utils/request';
 
-const defaultOptions = {
-    headers: {
-        'Content-Type': 'application/json',
-        'X-Requested-With': 'XMLHttpRequest', // Copied from sulu/sulu
-    },
-};
-
 class Requester {
-    handleResponse(response) {
+    requestErrorHandler = null;
+    defaultOptions = {
+        headers: {
```

Comment:
We should add credential to same origin else old browsers will not send the session cookie and the session get lost:

https://github.com/sulu/sulu/blob/ade35fff4cb8ced777c726dcb730c5c5f5720960/src/Sulu/Bundle/AdminBundle/Resources/js/services/Requester/Requester.js#L6

Reply:
ok is set in: https://github.com/sulu/SuluHeadlessBundle/blob/master/Resources/js-website/src/utils/request.js 👍